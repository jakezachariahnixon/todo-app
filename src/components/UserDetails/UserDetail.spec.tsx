import React from "react";
import renderer from "react-test-renderer";
import { cleanup } from "@testing-library/react";
import { UserDetails } from './index';

describe("UserDetails", () => {
    it("renders correctly", () => {
      const tree = renderer.create(<UserDetails></UserDetails>).toJSON();
      expect(tree).toMatchSnapshot();
    });
});
  
afterEach(cleanup);
