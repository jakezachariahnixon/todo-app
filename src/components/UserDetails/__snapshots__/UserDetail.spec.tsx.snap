// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserDetails renders correctly 1`] = `
<div
  className="c-icCVoz"
>
  <div
    className="c-RUvTe"
  >
    <img
      alt="avatar"
      src="userIcon.png"
      style={
        Object {
          "height": "80%",
          "width": "80%",
        }
      }
    />
  </div>
  <span>
    Jake Nixon
  </span>
</div>
`;
